src/lib/matching/getMatcherFunction.ts

```
@@ -0,0 +1,177 @@
+          isMatch = matchString(pattern, processedContent.content, options);
+          break;
+        case PatternType.REGEX:
+          isMatch = matchRegex(pattern, processedContent.content, options);
+          break;
+        case PatternType.GLOB:
+          // Glob matching inherently works on the preprocessed path string
+          isMatch = matchGlob(pattern, processedContent.content, options);
+          break;
+        default:
+          // Should be unreachable if PatternType enum is used correctly
+          const exhaustiveCheck: never = patternType;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)